exampleCourse/questions/element/checkbox/question.html

```diff
@@ -177,3 +177,29 @@
 
     </div>
 </div>
+
+<div class="card my-2">
+    <div class="card-header">
+        Part 7
+    </div>
+
+    <div class="card-body">
```

Comment:
Since we're adding a demo question for the checkbox feature, can we do something similar for the multiple choice element?

Reply:
added in 23a483a7d72641f42e5c8b9672ff8b50625ab8d0